### PR TITLE
Mainnet block time analysis

### PR DESCRIPTION
**Pull request by manav2401**

### CODE REVIEW

File: miner/miner.go [hunk 1]
```diff
@@ -68,9 +70,21 @@ type Miner struct {
 	stopCh   chan struct{}
 
 	wg sync.WaitGroup
+
+	log     *logrus.Logger
+	logFile *os.File
```
Comment (JekaMas) [inline miner/miner.go:75]: Can't we use standard ethereum logging?

File: consensus/bor/bor.go [hunk 2]
```diff
@@ -779,6 +780,7 @@ func (c *Bor) FinalizeAndAssemble(chain consensus.ChainHeaderReader, header *typ
 				return nil, err
 			}
 		}
+		log.Info("[Mining Analysis] Commit span and state-syncs", "time", time.Since(start1))
```
Comment (JekaMas) [inline consensus/bor/bor.go:783]: Could you add block number and hash(if present) everywhere where it's about mining?

Comment (JekaMas) [reply consensus/bor/bor.go:783]: Also could you use one field name for elapsed time? Here it is "time" and below it's 'elapsed'.

File: consensus/bor/bor.go [hunk 3]
```diff
@@ -1145,6 +1157,8 @@ func (c *Bor) fetchAndCommitSpan(
```
Comment (JekaMas) [inline consensus/bor/bor.go:1157]: That's a big piece of code to measure. Could you add intermediate points to measure?

File: consensus/bor/bor.go (hunk 2 above)
Comment (manav2401) [reply consensus/bor/bor.go:783]: Sure, making relevant changes. Thanks!

**Review by JekaMas — DISMISSED** [review]
LGTM!

**Pull request closed without merging**